tests/cupy_tests/creation_tests/test_from_data.py

```diff
@@ -551,7 +551,9 @@ def test_with_over_size_array(self):
             a = testing.shaped_random((size,), cupy, cupy.float64)
             b = cupy.asarray(DummyObjectWithCudaArrayInterface(a, 2, None))
             testing.assert_array_equal(a, b)
-        except cupy.cuda.memory.OutOfMemoryError:
+        # either numpy (np.core._exceptions._ArrayMemoryError) or cupy
+        # (cupy.cuda.memory.OutOfMemoryError) can raise
+        except MemoryError:
```

Comment:
`testing.slow` is marked to this test, so CI does not run this test case. Could you restore this change?

Reply:
This and the other tests both got run and showed OOM, which is why I correct them in this PR. I don't think `@testing.slow` matters here, see one of the CI log (not from this PR):
https://storage.googleapis.com/chainer-artifacts-pfn-public-ci/cupy-ci/69458/cupy_test_log.txt
(Need to search the test names because it's plain text and has no hyperlink.)

if you like I can take it away in this test since it doesn't make much sense indeed.

Reply:
I am looking at the pfnCI script and I don't understand why tests marked slow are still run...
https://github.com/cupy/cupy/blob/3c52ea06a4b21cbd94571d38999052740ca20410/.pfnci/windows/test.ps1#L95

Reply:
> if you like I can take it away in this test since it doesn't make much sense indeed.

I removed `@testing.slow` from this test.

Reply:
@leofang `-m "not slow"` option seems to be lost unexpectedly... I will fix it in https://github.com/cupy/cupy/pull/4819.

Reply:
Nice, thanks @asi1024 @kmaehashi! I'll make the other test below slow then.

Reply:
Could you restore `testing.slow` and remove try-except, to avoid skipping this test also for `slow` mode unexpectedly?

Reply:
@asi1024 I can restore `@testing.slow`, but are all CIs happy with large memory tests?

Reply:
No, it's not guaranteed to succeed to allocate large memory in all CIs. :cry:

Reply:
OK @asi1024 then I think it's best to keep try-except as is. I marked it slow in bd33816.

Reply:
We enabled "not slow" option in #4820, so I think we can remove try-except.

Reply:
Done in 3991318.

Reply:
Thanks!